tests/components/config/test_entity_registry.py

```diff
@@ -140,7 +141,7 @@ async def test_update_entity(hass, client):
     state = hass.states.get("test_domain.world")
     assert state is not None
     assert state.name == "before update"
-    assert state.attributes["icon"] == "icon:before update"
+    assert state.attributes[ATTR_ICON] == "icon:before update"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6212145)